src/base/bittorrent/cachestatus.cpp

```diff
@@ -50,7 +50,11 @@ qreal CacheStatus::readRatio() const
 
 int CacheStatus::jobQueueLength() const
 {
+#if LIBTORRENT_VERSION_NUM < 10100
     return m_nativeStatus.job_queue_length;
+#else
+    return m_nativeStatus.queued_jobs;
```

Comment:
@arvidn, which performance counters should I use as a replacement for the following cache_status fields?
* total_used_buffers
* blocks_read_hit
* blocks_read
* job_queue_length
* average_job_time
* average_job_time